sklearn/gaussian_process/gpr.py

```diff
@@ -142,9 +142,9 @@ def optimizer(obj_func, initial_theta, bounds):
     >>> kernel = DotProduct() + WhiteKernel()
     >>> gpr = GaussianProcessRegressor(kernel=kernel,
     ...         random_state=0).fit(X, y)
-    >>> gpr.score(X, y)
+    >>> gpr.score(X, y) # doctest: +ELLIPSIS
```

Comment:
if the tests are passing on these doctests, you don't need to add the `+ELLIPSIS` directive.

Reply:
There were multiple failings without the ellipsis, first on the local tests and when these passed, they failed the CI tests, so I think they were necessary almost everywhere

Reply:
Thomas just noted that the `ELLIPSIS` directive is a default in our settings. Therefore we don't need it.